test/unit/mdc-drawer/dismissible.foundation.test.js

```diff
@@ -45,8 +45,8 @@ test('exports cssClasses', () => {
 
 test('defaultAdapter returns a complete adapter implementation', () => {
   verifyDefaultAdapter(MDCDismissibleDrawerFoundation, [
-    'hasClass', 'addClass', 'removeClass', 'computeBoundingRect', 'setStyleAppContent',
-    'addClassAppContent', 'removeClassAppContent', 'isRtl', 'notifyClose', 'notifyOpen',
+    'hasClass', 'addClass', 'removeClass', 'eventTargetHasClass', 'computeBoundingRect', 'saveFocus', 'restoreFocus',
```

Comment:
Also add a test for `computeBoundingRect` to the component unit test module for the vanilla implementation